clubs/serializers.py

```diff
@@ -172,11 +173,13 @@ def validate(self, data):
         return data
 
     def save(self):
-        if 'club' not in self.validated_data:
-            club_code = self.context['view'].kwargs.get('club_code')
-            if club_code is None:
-                club_code = self.context['view'].kwargs.get('pk')
-            self.validated_data['club'] = Club.objects.get(code=club_code)
+        """
+        Fill in club field from URL instead of from request data.
```

Comment:
An alternative that might save us a DB query would be to make the `club` field `read_only=True` in the serializer, so API calls wouldn't be able to change it.

Reply:
The club field actually isn't in the list of valid fields, so the user cannot set this field anyways.

I don't think the query is avoidable, the queryless alternative would be to use the primary key directly, which the user doesn't pass to us (`code` != `id`).